lib/Migration/Version2900Date20250718065820.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\OpenProject\Migration;
+
+use Closure;
+use OCA\OpenProject\AppInfo\Application;
+use OCA\OpenProject\Service\OpenProjectAPIService;
+use OCP\DB\ISchemaWrapper;
+use OCP\IConfig;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+use Override;
+
+class Version2900Date20250718065820 extends SimpleMigrationStep {
```

Comment:
```suggestion
class Version2910Date20250718065820 extends SimpleMigrationStep {
```

IMO, our app version is 2.9.1 so according to it migration file named like this :up: ,
isn't it?